packages/dart/npt_flutter/changelog.md

```diff
@@ -1,3 +1,12 @@
+## 1.7.0+24
+- 
```

Comment:
You forgot to delete the "-". I can be easily corrected in the future so I'll go ahead and approve.